rules/S6885/java/metadata.json

```
@@ -0,0 +1,23 @@
+{
+  "title": "Use built-in \"Math.clamp\" methods",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
+  },
+  "tags": [ "java21"
+  ],
```

Just for readability, you can either inline the closing bracket, or put the tag on a new line
```suggestion
  "tags": [
  "java21"
  ],
```

---

:+1: